src/Resolver.js

```diff
@@ -73,10 +74,8 @@ class Resolver {
     }
 
     if (!resolved) {
-      let err = new Error(
-        "Cannot find module '" + input + "' from '" + dir + "'"
-      );
-      err.code = 'MODULE_NOT_FOUND';
+      let err = new Error(`Cannot find module '${input}' from '${dir}'`);
+      err.code = isRelativeFile ? 'MODULE_NOT_FOUND' : 'NODE_MODULE_NOT_FOUND';
```

Comment:
unfortunately, this error code is standard as part of the node require resolution algorithm, so I don't think we can change it https://nodejs.org/api/modules.html#modules_file_modules

Reply:
Yeah, probably not a good idea in that case